CONTRIBUTING.md

```diff
@@ -199,12 +199,18 @@ The build process is divided into stages, so you can invoke them as needed:
   (running this script requires that you installed the [toolchains](#Toolchains) for all target languages on your
   system).
 
-### Partial build tools
+##### Partial builds
 
-There are also two useful scripts in the `scripts` directory that can help you build part of the repo:
+When working on an individual module, the `build.sh` can be used to build that module specifically along with any related modules. The following is a list of scripts to build a module under development:
 
-- __`scripts/buildup`__: builds the current module and all of its dependencies (in topological order).
-- __`scripts/builddown`__: builds the current module and all of its consumers (in topological order).
+- __`build.sh --scope [<module-name>]`__: builds the module with the specified name.
```

Comment:
Please indicate that `--scope` is optional if your working directory is a specific module. I think that would be the more common use case for most people:

```shell
build.sh --up [-- scope MODULE]
```